core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -456,7 +651,8 @@ public String buildKeyName(long keynum) {
 	public boolean doInsert(DB db, Object threadstate)
 	{
 		int keynum=keysequence.nextInt();
-		String dbkey = buildKeyName(keynum);
+		//String dbkey = buildKeyName(keynum);
```

Comment:
if you're going to change these lines, just change them rather than commenting them out. we can always use git to retrieve them.